test_conformance/extensions/cl_khr_command_buffer/negative_command_buffer_svm_mem.cpp

```diff
@@ -0,0 +1,413 @@
+        error = clCommandSVMMemFillKHR(
+            command_buffer, nullptr, svm_in_mem(), &pattern_1, sizeof(cl_char),
+            data_size(), 1, invalid_sync_points[0], nullptr, nullptr);
+
+        test_failure_error_ret(error, CL_INVALID_SYNC_POINT_WAIT_LIST_KHR,
+                               "clCommandSVMMemFillKHR should return "
+                               "CL_INVALID_SYNC_POINT_WAIT_LIST_KHR",
+                               TEST_FAIL);
+
+
+        error = clCommandSVMMemcpyKHR(command_buffer, nullptr, svm_out_mem(),
+                                      svm_in_mem(), data_size(), 1, nullptr,
+                                      nullptr, nullptr);
+
+        test_failure_error_ret(error, CL_INVALID_SYNC_POINT_WAIT_LIST_KHR,
+                               "clCommandSVMMemcpyKHR should return "
+                               "CL_INVALID_SYNC_POINT_WAIT_LIST_KHR",
+                               TEST_FAIL);
+
+        error = clCommandSVMMemFillKHR(command_buffer, nullptr, svm_in_mem(),
+                                       &pattern_1, sizeof(cl_char), data_size(),
+                                       1, nullptr, nullptr, nullptr);
+
+        test_failure_error_ret(error, CL_INVALID_SYNC_POINT_WAIT_LIST_KHR,
+                               "clCommandSVMMemFillKHR should return "
+                               "CL_INVALID_SYNC_POINT_WAIT_LIST_KHR",
+                               TEST_FAIL);
+
+
+        cl_sync_point_khr point;
+        error = clCommandBarrierWithWaitListKHR(command_buffer, nullptr, 0,
+                                                nullptr, &point, nullptr);
+        test_error(error, "clCommandBarrierWithWaitListKHR failed");
+        cl_sync_point_khr* sync_points[] = { &point };
+
+        error = clCommandSVMMemcpyKHR(command_buffer, nullptr, svm_out_mem(),
+                                      svm_in_mem(), data_size(), 0,
+                                      sync_points[0], nullptr, nullptr);
+
+        test_failure_error_ret(error, CL_INVALID_SYNC_POINT_WAIT_LIST_KHR,
+                               "clCommandSVMMemcpyKHR should return "
+                               "CL_INVALID_SYNC_POINT_WAIT_LIST_KHR",
+                               TEST_FAIL);
+
+        error = clCommandSVMMemFillKHR(command_buffer, nullptr, svm_in_mem(),
+                                       &pattern_1, sizeof(cl_char), data_size(),
+                                       0, sync_points[0], nullptr, nullptr);
+
+        test_failure_error_ret(error, CL_INVALID_SYNC_POINT_WAIT_LIST_KHR,
+                               "clCommandSVMMemFillKHR should return "
+                               "CL_INVALID_SYNC_POINT_WAIT_LIST_KHR",
+                               TEST_FAIL);
+
+
+        return CL_SUCCESS;
+    }
+
+    bool Skip() override
+    {
+        return BasicSVMCommandBufferTest::Skip()
+            || is_extension_available(device,
+                                      "cl_khr_command_buffer_multi_device");
```

Comment:
Even with the multidevice extension I think this would still be an error, so don't need to skip with the presence of this extension.

Reply:
@EwanC I will remove Skip() for this test.